frontend/src/features/notifications/components/NotificationInformationModal.tsx

```diff
@@ -0,0 +1,381 @@
+/*
+Copyright (c) 2024 Volkswagen AG
+Copyright (c) 2024 Contributors to the Eclipse Foundation
+
+See the NOTICE file(s) distributed with this work for additional
+information regarding copyright ownership.
+
+This program and the accompanying materials are made available under the
+terms of the Apache License, Version 2.0 which is available at
+https://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Input, PageSnackbar, PageSnackbarStack, Textarea } from '@catena-x/portal-shared-components';
+import { DateTime } from '@components/ui/DateTime';
+import { Close, Save } from '@mui/icons-material';
+import { Autocomplete, Box, Button, Dialog, DialogTitle, FormLabel, Grid, InputLabel, Stack } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { LabelledAutoComplete } from '@components/ui/LabelledAutoComplete';
+import { postDemandAndCapacityNotification } from '@services/demand-capacity-notification';
+import { Notification } from '@models/types/data/notification';
+import { EFFECTS } from '@models/constants/effects';
+import { useAllPartners } from '@hooks/useAllPartners';
+import { LEADING_ROOT_CAUSE } from '@models/constants/leading-root-causes';
+import { STATUS } from '@models/constants/status';
+import { DemandCapacityNotification } from '@models/types/data/demand-capacity-notification';
+import { Site } from '@models/types/edc/site';
+import { useSites } from '@features/stock-view/hooks/useSites';
+import { usePartnerMaterials } from '@hooks/usePartnerMaterials';
```

Comment:
As a material can have both directions, do we then send the message twice? The reselling is pretty strange in that case. Maybe we don't need to care about.

Reply:
The way I see it, direction is implied by `effect`. Sending a message with the same effect in both directions makes little sense and if the effect is different then it needs to be 2 notifications in any case.

I'm gladly up for discussing this topic if this logic isn't sufficient.

Reply:
Ah fair, didn't had that in mind. Thanks for explaining!